server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v64/DbVersion64.java

```diff
@@ -28,6 +28,11 @@
   public void addSteps(MigrationStepRegistry registry) {
     registry
       .add(1600, "Add Projects.TAGS", AddTagsToProjects.class)
```

Comment:
out of scope, but could rename the description of migration 1600 by having upper-case "Add PROJECTS.TAGS" ? Thanks